Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -223,7 +223,7 @@ internal static void ActivateLocalClientScene()
             if (identity != null && identity.observers != null)
             {
                 // get writer from pool
-                NetworkWriter writer = NetworkWriterPool.GetWriter();
+                PooledNetworkWriter writer = NetworkWriterPool.GetWriter();
```

Comment:
should we put those into 'using' already?
otherwise recycle is called twice unless I am missing something